.pylintrc

```diff
@@ -0,0 +1,577 @@
+[THINGS WE CARE ABOUT]
+
+[FORMAT]
+
+# Expected format of line ending, e.g. empty (any line ending), LF or CRLF.
+expected-line-ending-format=
+# Naming style matching correct class names.
+class-naming-style=PascalCase
+
+# Maximum number of nested blocks for function / method body
+max-nested-blocks=5
+
+# Maximum number of characters on a single line.
+max-line-length=90
+
+# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1
+# tab).
+indent-string='    '
+
+# Number of spaces of indent required inside a hanging or continued line.
+indent-after-paren=4

```

Comment:
Do we want indention to be with 2 spaces or 4? I tend to prefer 4 as it helps to differentiate blocks better, which is especially important in python.

Reply:
4 for Python. Definitely.

Reply:
But wait, maybe 2 could be okay for a continued line? To differentiate a continued line from a new block?

Reply:
That sounds great to me honestly. I never know how much to space a continued line but it makes it easy to differentiate.

Reply:
Also just realized that's the property I was commenting on originally. The general tab size is another property, so I'm gonna find that property and add it to the top as well